### PR TITLE
Changes to the Readme.

### PR DESCRIPTION
**Pull request by WatsonCIQ**
- Table of contents
- Simplified installation
- better FAQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,33 +39,39 @@ The Finsemble FDC3 implementation is comprised of four parts:
...
+
+<a href="preload" title="#preload">**Preload:**</a>
+The FDC3 preload is required to enable the FDC3 Client in your components to use the FDC3 Client API. Adding the preload is as simple as adding the preload to the preload section (see the config example below).
 
 **Toolbar Icon URL:**
-The icon url is used by the intent resolver and will be used to display the logo of the your component. This URL can use the $applicationRoot or can be an external URL. Supported formats: jpg, png, svg
+The icon url is used by the intent resolver to display the logo of the your component. This URL can use the $applicationRoot or can be an external URL. Supported formats: jpg, png, svg
 
 **FDC3:**
 This section includes the intents and context that your
```
Comment (kriswest) [inline README.md:77]:
```suggestion
An additional section in your component config, `foreign.services.fdc3`, is used to specify intents and associated context types that your
```
probably should also remove the line break here

File: README.md [hunk 2]
```diff
@@ -87,22 +112,13 @@ This section includes the intents and context that your
 ```
 
 
-## Service
-The service houses most of the logic for the FDC3 integration support but you will not need knowledge of this section unless you are contributing to the FDC3 Finsemble codebase.
-
+### FDC3 in Finsemble Services
+Please see our [exampleFDC3Service](./src/services/testFDC3) for how to structure your service to work with FDC3.
```
Comment (kriswest) [inline README.md:116]:
```suggestion
The Finsemble FDC3 client may also be used in Finsemble services, where it is imported into the service implementation. Please see our [exampleFDC3Service](./src/services/testFDC3) for an example of how to structure your service to work with FDC3.
```

File: README.md [hunk 3]
```diff
@@ -121,7 +137,7 @@ To get started with the API you will need to use the Finsemble FDC3 DesktopAgent
 
 <br/>
 
-### Context
+### Context:
```
Comment (kriswest) [inline README.md:140]:
```suggestion
### Context
```

File: README.md [hunk 4]
```diff
@@ -87,22 +112,13 @@ This section includes the intents and context that your
...
-## API:
-
-To get started with the API you will need to use the Finsemble FDC3 DesktopAgent -
-`fdc3`
+## API
 
-**The code snippets below assume that you prepend the code with the desktop agent code snippet above.*
-
-### App
+### App:
```
Comment (kriswest) [inline README.md:121]:
```suggestion
### App
```

File: README.md [hunk 5]
```diff
@@ -160,7 +176,7 @@ To get started with the API you will need to use the Finsemble FDC3 DesktopAgent
 
 <br/>
 
-### Intents
+### Intents:
```
Comment (kriswest) [inline README.md:179]:
```suggestion
### Intents
```

File: README.md [hunk 6]
```diff
@@ -205,9 +221,9 @@ context => { //do something here  }
 
 <br/>
 
-### Channel
+### Channels:
```
Comment (kriswest) [inline README.md:224]:
```suggestion
### Channels
```

File: README.md [hunk 7]
```diff
@@ -87,22 +112,13 @@ This section includes the intents and context that your
...
 
-## Client
 
-_The FDC3 Client is added in via preload, you now have access to this client as if it were any other Finsemble client._
-
-## API:
-
-To get started with the API you will need to use the Finsemble FDC3 DesktopAgent -
-`fdc3`
+## API
```
Comment (kriswest) [inline README.md:120]:
```suggestion
Please note that the FINOS FDC3 website provides documentation for the [Desktop Agent](https://fdc3.finos.org/docs/1.1/api/ref/DesktopAgent) and [Channel](https://fdc3.finos.org/docs/1.1/api/ref/Channel) APIs. The main API calls are also detailed here for convenience.
```

File: README.md [hunk 8]
```diff
@@ -205,9 +221,9 @@ context => { //do something here  }
 
 <br/>
 
-### Channel
+### Channels:
 
-A channel has this interface:
+Example of the FDC3 Channel Interface:
```
Comment (kriswest) [inline README.md:226]:
```suggestion
The DesktopAgent API will allow you to join, listen to and broadcast context to a single 'channel' via its API. However, you can also interact directly with FDC3 channels, which have the following interface:
```

File: README.md [hunk 9]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
+>
+FDC3 1.1 introduces the Channels API. You can connect your components to a channel either programmatically or user-led via the Finsemble Linker.
+Channels then allow you to share context between them. A single application (component) can only join one channel at a time e.g it cannot join the purple and the yellow channel, if the user tries to do this they will be removed from the last channel joined. You can however programmatically listen to as many channels as you like from your application.
 Many applications can be joined to a single-channel e.g App A, B and C can all be joined by the purple channel.
 
-### What do I need to do to enable the channels so that my end users can start using it?
+> What do I need to do to enable the channels so that my end users can start using it?
+>
 Finsemble allows you to use the Linker Channels. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```
Comment (kriswest) [inline README.md:333]:
```suggestion
Finsemble includes user interface for selecting built-in (or system) colour channels, normally used with the Finsemble Linker API. The FInsemble FDC3 implementation, if preloaded into a component, will take over the Linker UI allowing you to use it select FDC3 system channels instead. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```

@WatsonCIQ can you confirm if they also need to enable the linker in their config or whether the preload does this for you please.

File: README.md [hunk 10]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
 
-### How do I know what channels I have access to?
+> How do I know what channels I have access to?
+>
 System channels (including a global channel) are listed by doing the following:
```
Comment (kriswest) [inline README.md:337]:
```suggestion
You can list the available System channels (including a global channel) by doing the following:
```

File: README.md [hunk 11]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
+const listener = fdc3.addContextListener(context => { ... });
 
-// listener for a specific type  
-const contactListener = fdc3.addContextListener('fdc3.contact', contact => { ... }); 
+// listener for a specific type
+const contactListener = fdc3.addContextListener('fdc3.contact', contact => { ... });
 ```
 
 If you want to listen to a specific channel you need to get it first like this:
 ```
```
Comment (kriswest) [inline README.md:387]:
```suggestion
```javascript
```

File: README.md [hunk 12]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
+const instrument = {
+  type: 'fdc3.instrument',
+  id: { ticker: 'MSFT' }
+};
 fdc3.broadcast(instrument);
 ```
 
-If you want to send context to a channel you are not joined to you need to get the channel first. 
+If you want to send context to a channel you are not joined to you need to get the channel first.
 ```
```
Comment (kriswest) [inline README.md:357]:
```suggestion
```javascript
```

File: README.md [hunk 13]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
-### Am I limited to just your Finsemble Linker channels?
+> Am I limited to just your Finsemble Linker channels?
+>
 No. You can add as many channels as you like, try it by using `const Channel = fdc3.getOrCreateChannel("myChannel")`
 
-### You mentioned programmatically sending context (data) over channels how do I do that?
+> You mentioned programmatically sending context (data) over channels how do I do that?
+>
 If you want to send context via the channel that you are currently joined to you can do the following:
 ```
```
Comment (kriswest) [inline README.md:348]:
```suggestion
```javascript
```

File: README.md [hunk 14]
```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
...
 
-### I want to send an instrument (MSFT) but I want the user to choose the application they want to display it with.
+> I want to send an instrument (MSFT) but I want the user to choose the application they want to display it with, how would do I that?
 
 This is where you can raise an intent. Once the intent has been raised it will show all the apps capable of dealing with your intent in the intent resolver. This works in a similar fashion to your phone when you want to share a link or open a calendar invite and it asks which application you would like to use.
-To do this you need two things 
+To do this you need two things
 1) A Finsemble component that has been set up to accept your intent (you can read the section on config above to do this).
 2) Your component to have a button or event to run the following code
 ```
```
Comment (kriswest) [inline README.md:302]:
```suggestion
```javascript
```

File: README.md [hunk 15]
```diff
@@ -259,12 +275,14 @@ interface Channel {
...
 There is a simple command for this! `fdc3.open("ChartIQ")`
 
-### My Chart opens but now I need to send instrument data to it.
-The fdc3.open call allows you to send context with it too. 
-Here is an example of opening the chart and sending the GOOGL (Google) ticker symbol to it. 
+> My Chart opens but now I need to send instrument data to it, what do I need to do next?
+
+The fdc3.open call allows you to send context with it too.
+Here is an example of opening the chart and sending the GOOGL (Google) ticker symbol to it.
 ```
```
Comment (kriswest) [inline README.md:286]:
```suggestion
```javascript
```

File: README.md [hunk 16]
```diff
@@ -205,9 +221,9 @@ context => { //do something here  }
 
 <br/>
 
-### Channel
+### Channels:
 
-A channel has this interface:
+Example of the FDC3 Channel Interface:
 ```
```
Comment (kriswest) [inline README.md:227]:
```suggestion
```javascript
```

File: README.md [hunk 17]
```diff
@@ -275,105 +298,114 @@ fdc3.open("ChartIQ",{
...
 
-### Now I have sent context (data) how do I listen for it?<a name="context-listener"></a>
+> Now I have sent context (data) how do I listen for it?
 
-It depends on if you want to listen for context on the channel you are joined to or a different channel(s). 
+It depends on if you want to listen for context on the channel you are joined to or a different channel(s).
 
-If you are joined to a channel you can use the following: 
+If you are joined to a channel you can use the following:
 ```
```
Comment (WatsonCIQ) [inline README.md:383]:
```suggestion
```javascript
```

File: README.md [hunk 18]
```diff
@@ -20,36 +38,48 @@ The Finsemble FDC3 implementation is comprised of four parts:
...
+- Ensure that you have a Toolbar Icon URL set 
+- Add FDC3 specific configuration
 
-## Component Config & Preload
-The Finsemble config file allows Finsemble to understand that your application is built with FDC3 capability.
 
-### Config Sections:
-**Preload:**
-The preload is needed to use the FDC3 Client API, add this to the preload section.
+**Preload**
```
Comment (kriswest) [inline README.md:75]:
```suggestion
##### Preload
```

File: README.md [hunk 19]
```diff
@@ -20,36 +38,48 @@ The Finsemble FDC3 implementation is comprised of four parts:
...
+To add the FDC3 preload to your component, set `component.preload` in its configuration to the path to the built preload script, e.g. `"$applicationRoot/preloads/FDC3Client.js"`, or if using multiple preload scripts add it to an array of their paths (see the config example below).
 
-**Toolbar Icon URL:**
-The icon url is used by the intent resolver and will be used to display the logo of the your component. This URL can use the $applicationRoot or can be an external URL. Supported formats: jpg, png, svg
+**Toolbar Icon URL**
```
Comment (kriswest) [inline README.md:78]:
```suggestion
##### Toolbar Icon URL
```

File: README.md [hunk 20]
```diff
@@ -20,36 +38,48 @@ The Finsemble FDC3 implementation is comprised of four parts:
...
+**Note:** The watch script will continue to monitor this project for file changes. If you do not need it to, it  can be stopped once all the files have been copied to the seed project (which takes approx. 30 seconds).
+6) Follow the instructions on how to add the FDC3 to your [Component](###-FDC3-in-Finsemble-Components) / [Service](###FDC3-in-Finsemble-Services):
+
+## Using FDC3 in Finsemble
+The Finsemble implementation of the FDC3 DesktopAgent and Channels APIs can be used in both Finsemble Components and Services.
+
+### FDC3 in Finsemble Components
+The FDC3 client is most easily made available to your components via preload, which will give your component access to it as if it were any other Finsemble client. It is anticipated that later versions of Finsemble will include the FDC3 client by default.
+
+**Component Config**
```
Comment (kriswest) [inline README.md:68]:
```suggestion
#### Component Config
```
Ended up running into next line, better stick to heading styles

File: README.md [hunk 21]
```diff
@@ -20,36 +38,48 @@ The Finsemble FDC3 implementation is comprised of four parts:
...
+To add the FDC3 preload to your component, set `component.preload` in its configuration to the path to the built preload script, e.g. `"$applicationRoot/preloads/FDC3Client.js"`, or if using multiple preload scripts add it to an array of their paths (see the config example below).
 
-**Toolbar Icon URL:**
-The icon url is used by the intent resolver and will be used to display the logo of the your component. This URL can use the $applicationRoot or can be an external URL. Supported formats: jpg, png, svg
+**Toolbar Icon URL**
+The icon url is used by the intent resolver to display the logo of your component. This URL can use the `$applicationRoot` variable to set a local path or can be an external URL. Supported formats: `.jpg`, `.png`, `.svg`.
 
-**FDC3:**
-This section includes the intents and context that your
+**FDC3 configuration**
```
Comment (kriswest) [inline README.md:81]:
```suggestion
##### FDC3 configuration
```

File: README.md [hunk 22]
```diff
@@ -275,105 +298,114 @@ fdc3.open("ChartIQ",{
...
 Many applications can be joined to a single-channel e.g App A, B and C can all be joined by the purple channel.
 
-### What do I need to do to enable the channels so that my end users can start using it?
-Finsemble allows you to use the Linker Channels. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
+> What do I need to do to enable the channels so that my end users can start using it?
+>
+Finsemble includes user interface for selecting built-in (or system) colour channels, normally used with the Finsemble Linker API. The FInsemble FDC3 implementation, if preloaded into a component, will take over the Linker UI allowing you to use it select FDC3 system channels instead. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```
Comment (kyle-ciq) [inline README.md:338]:
```suggestion
Finsemble includes user interface for selecting built-in (or system) color channels, normally used with the Finsemble Linker API. The Finsemble FDC3 implementation, if preloaded into a component, will take over the Linker UI allowing you to use it select FDC3 system channels instead. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```

**Pull request merged**